example/src/classes/util/default-builder.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Пример наследования классов.
+ * Базовый строитель реализует методы, которые присущи любому классу строителя, который
+ * наследуется от него. Дочерний строитель (унаследованный от данного класса) к базовым
+ * методам может добавить, при необходимости, собственные. Дочерний строитель может
+ * переопределить реализацию какого-либо из методов родителя.
+ */
+export default class DefaultBuilder {
```

Comment:
We should delete this class.
Builder pattern is useful in languages like Java, where we can't pass many arguments in a row, but in js we can pass an object with all props we need.
Methods of this class can be used in our base class for all components, but they shouldn't return `this`.